elasticapm/contrib/django/client.py

```diff
@@ -66,9 +66,9 @@ def get_client(client=None):
 
 
 def get_client_default_config():
-    config = getattr(django_settings, 'ELASTIC_APM', {})
     return dict(
-        include_paths=set(config.get('INCLUDE_PATHS', [])) | _get_installed_apps_paths(),
+        include_paths=_get_installed_apps_paths(),
+        exclude_paths={'django'}
```

Comment:
L71 is not documented, right? default `exclude_paths` shouldn't be "depending on framework"?

Reply:
Good catch! I mention in the `include_paths` section that for Django, `django.contrib` apps are excluded (which is caused by this exclude), but I should also mention it in the `exclude_paths` setting